packages/manager/tools/component-rollup-config/src/plugins/translation-ui-router.ts

```diff
@@ -51,6 +51,7 @@ export = (opts: any = {}) => {
               props.filter(
                 (item) =>
                   item &&
+                  item.key &&
```

Comment:
Maybe you can use optional chaining operator:
```suggestion
                  item.key?.name === 'translations'
```